luaui/Widgets/gui_camera_anchors.lua

```diff
@@ -30,21 +31,20 @@ function SetCameraAnchor(_, _, args)
 	local anchorId = args[1]
 	local cameraState = GetCameraState()
```

Comment:
Actually, can you confirm again that you are restoring the zoom level at the time of anchor set? `GetCameraState()` might differ with `GetCameraState(false)` but I'm not really used to camera callouts.

Reply:
Yeah when I switch anchors it changes the zoom but also rotation of the camera!

Reply:
Nice, glad to hear that :clap: